tests/test_pca.py

```diff
@@ -780,3 +780,15 @@ def test_unknown_shapes_n_components_larger_than_num_rows(solver):
     else:
         with pytest.raises(ValueError, match="n_components is too large"):
             pca.fit(X)
+
+
+@pytest.mark.parametrize("input_type", ["array", "dataframe"])
+def test_pca_sklearn_inputs(input_type):
+    a = dd.PCA()
+
+    assert isinstance(X, np.ndarray)
+    _X = X if input_type != "dataframe" else pd.DataFrame(X)
```

Comment:
To avoid this string comparision, I'd recommend using `input_type=[np.array, pd.DataFrame])` above and `X = input_type(X)`